service/resource/project/resource.go

```diff
@@ -185,9 +193,19 @@ func (r *Resource) ProcessCreateState(obj, createState interface{}) error {
 		r.logger.Log("debug", "creating projects in the Kubernetes cluster")
 
 		for _, p := range projectsToCreate {
-			err := r.installer.Install(installerspec.Project{Name: p.Name, Ref: p.Ref})
-			if err != nil {
-				return microerror.Mask(err)
+			instErr := r.installer.Install(installerspec.Project{Name: p.Name, Ref: p.Ref})
+			if instErr != nil {
+				notiErr := r.notifier.Failed(notifierspec.Project{Name: p.Name, Ref: p.Ref}, instErr.Error())
+				if notiErr != nil {
+					return microerror.Mask(notiErr)
+				}
+
+				return microerror.Mask(instErr)
+			}
+
+			notiErr := r.notifier.Success(notifierspec.Project{Name: p.Name, Ref: p.Ref})
+			if notiErr != nil {
+				return microerror.Mask(notiErr)
```

Comment:
This code is duplicated below. Good enough for me now. Before I improve this somehow I would rather like to improve the difference between installing and upgrading charts via helm.